composer.json

```diff
@@ -251,38 +254,38 @@
         "drupal/slick_views": "1.0-rc2",
         "drupal/libraries": "3.x-dev",
         "drupal/datalayer": "1.x-dev",
-        "drupal/optimizely": "3.0",
-        "drupal/google_tag": "1.0",
-        "drupal/google_analytics": "2.2",
-        "drupal/simple_sitemap": "2.9",
-        "drupal/redirect": "1.0-beta1",
-        "drupal/metatag": "1.3",
+        "drupal/optimizely": "3.1",
+        "drupal/google_tag": "1.1",
+        "drupal/google_analytics": "2.3",
+        "drupal/simple_sitemap": "2.12",
+        "drupal/redirect": "1.3",
+        "drupal/metatag": "1.7",
         "drupal/scheduler": "1.0",
-        "drupal/webform": "5.0-beta24",
+        "drupal/webform": "5.0-rc16",
         "drupal/captcha": "1.0-beta1",
-        "drupal/recaptcha": "2.2",
-        "drupal/rabbit_hole": "1.0-beta3",
-        "drupal/admin_toolbar": "1.20",
+        "drupal/recaptcha": "2.3",
+        "drupal/rabbit_hole": "1.0-beta4",
+        "drupal/admin_toolbar": "1.24",
         "drupal/panelbutton": "1.1",
         "drupal/colorbutton": "1.1",
-        "drupal/better_exposed_filters": "3.0-alpha2",
+        "drupal/better_exposed_filters": "3.0-alpha5",
         "drupal/custom_formatters": "3.0-alpha1",
-        "drupal/token_filter": "1.0-beta1",
-        "drupal/views_field_formatter": "1.5",
+        "drupal/token_filter": "1.1",
+        "drupal/views_field_formatter": "1.8",
         "drupal/tzfield": "1.x-dev",
         "drupal/views_data_export": "1.0-beta1",
-        "drupal/lndr": "1.11",
+        "drupal/lndr": "1.15",
```

Comment:
~1.15
no need to stick to exact version for this one

Reply:
Fixed.